pagopa/io-app
# chore: [IOBP-176] Add new pictograms in ID Pay onboarding failure screen

## Short description
This PR adds the correct pictograms in ID Pay onboarding failure screen

## List of changes proposed in this pull request
- Updated `@pagopa/io-app-design-system` to v 1.18.0
- Replaced pictograms in `FailureScreen.tsx`

## How to test
Start a new ID Pay onboarding from the ID Pay playground and check all the various failure screens.
